app/models/form526_submission.rb

```diff
@@ -55,7 +55,7 @@ def start
         'submission_id' => id
       )
       jids = workflow_batch.jobs do
-        DisabilityCompensationFastTrackJob.perform_async(id, get_full_name)
```

Comment:
Should this be renamed to FastTrack::DisabilityCompensationJob, or should that module be renamed? I think I prefer this name to the other.

Reply:
eek - It should be `FastTrack::DisabilityCompensationJob` and I should look into why the spec didn't hit that code. 

I like the old name too. Do you have any ideas for naming the module?

Reply:
I commented on the wrong side of the diff here, sorry - I meant that I prefer `DisabilityCompensationFastTrack::Job`, but that's not a passionate preference.

Reply:
It should be fixed now .. I mean it the test is now passing etc.